2015-11-03 17:45:54 Error: [Cake\Routing\Exception\MissingControllerException] Controller class CakePHP could not be found.
Exception Attributes: array (
  'class' => 'CakePHP',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/cakePHP/
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:36:54 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Amin could not be found.
Exception Attributes: array (
  'class' => 'Amin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Amin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:37:00 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:41:49 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:41:53 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:41:53 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:41:54 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:42:00 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin.php could not be found.
Exception Attributes: array (
  'class' => 'Admin.php',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin.php
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:45:24 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin.php could not be found.
Exception Attributes: array (
  'class' => 'Admin.php',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin.php
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:45:26 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin.php could not be found.
Exception Attributes: array (
  'class' => 'Admin.php',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin.php
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:45:36 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:46:30 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:49:13 Error: Fatal Error (1): Call to a member function allow() on boolean in [C:\xampp\htdocs\CakePHP\src\Controller\AdminController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-03 18:49:14 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on boolean
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-03 18:49:14 Error: Fatal Error (1): Call to a member function allow() on boolean in [C:\xampp\htdocs\CakePHP\src\Controller\AdminController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-03 18:49:14 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on boolean
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-03 18:56:02 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:56:03 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Admin could not be found.
Exception Attributes: array (
  'class' => 'Admin',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/Admin
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:56:09 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Users could not be found.
Exception Attributes: array (
  'class' => 'Users',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:56:14 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Users could not be found.
Exception Attributes: array (
  'class' => 'Users',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-03 18:56:49 Error: Fatal Error (1): Call to a member function allow() on boolean in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-03 18:56:49 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on boolean
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 13:49:49 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 13:49:50 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 13:51:34 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 13:51:35 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 13:58:25 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 13:58:26 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 13:58:33 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 13:58:34 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:00:33 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:00:34 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/Users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:03:04 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:03:04 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/Users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:03:41 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:03:42 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/Users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:03:53 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:03:54 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:04:43 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:04:44 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:04:46 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:04:46 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:04:46 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:04:47 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:04:47 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:04:47 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:04:54 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:04:54 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:06:15 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 13]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:06:16 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 13)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:07:33 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:07:34 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:08:28 Error: [BadMethodCallException] Unknown method "allow"
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php(14): Cake\ORM\Table->__call('allow', Array)
#1 C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php(14): App\Model\Table\UsersTable->allow('add')
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Event\EventManager.php(385): App\Controller\UsersController->beforeFilter(Object(Cake\Event\Event))
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Event\EventManager.php(355): Cake\Event\EventManager->_callListener(Array, Object(Cake\Event\Event))
#4 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Event\EventDispatcherTrait.php(78): Cake\Event\EventManager->dispatch(Object(Cake\Event\Event))
#5 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(478): Cake\Controller\Controller->dispatchEvent('Controller.init...')
#6 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(109): Cake\Controller\Controller->startupProcess()
#7 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#8 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#9 {main}


2015-11-04 14:08:53 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:08:54 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:08:54 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:08:54 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:08:54 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:08:55 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:10:14 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:10:15 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:10:15 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:10:15 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:10:16 Error: Fatal Error (1): Call to a member function allow() on a non-object in [C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php, line 14]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:10:16 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:10:16 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:10:16 Error: [Cake\Error\FatalErrorException] Call to a member function allow() on a non-object
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 14)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:14:49 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Add could not be found.
Exception Attributes: array (
  'class' => 'Add',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/add
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 14:17:26 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Add could not be found.
Exception Attributes: array (
  'class' => 'Add',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/add
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 14:17:35 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\login.ctp',
)
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 14:21:59 Error: Fatal Error (1): Call to a member function create() on a non-object in [C:\xampp\htdocs\CakePHP\src\Template\Users\login.ctp, line 4]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:22:00 Error: [Cake\Error\FatalErrorException] Call to a member function create() on a non-object
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 4)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:22:11 Error: Fatal Error (1): Call to a member function create() on a non-object in [C:\xampp\htdocs\CakePHP\src\Template\Users\login.ctp, line 4]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:22:12 Error: [Cake\Error\FatalErrorException] Call to a member function create() on a non-object
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 4)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 14:22:23 Error: Fatal Error (1): Call to a member function create() on a non-object in [C:\xampp\htdocs\CakePHP\src\Template\Users\login.ctp, line 4]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 14:22:24 Error: [Cake\Error\FatalErrorException] Call to a member function create() on a non-object
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 4)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 16:58:40 Error: Fatal Error (1): Call to a member function create() on a non-object in [C:\xampp\htdocs\CakePHP\src\Template\Users\login.ctp, line 4]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 16:58:41 Error: [Cake\Error\FatalErrorException] Call to a member function create() on a non-object
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 4)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 16:58:55 Error: Fatal Error (1): Call to a member function create() on a non-object in [C:\xampp\htdocs\CakePHP\src\Template\Users\login.ctp, line 4]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 16:58:56 Error: [Cake\Error\FatalErrorException] Call to a member function create() on a non-object
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Call to a membe...', 'C:\\xampp\\htdocs...', 4)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 16:59:30 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\login.ctp',
)
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 16:59:55 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\login.ctp',
)
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 17:10:32 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\login.ctp',
)
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 17:14:04 Error: [Cake\Controller\Exception\MissingActionException] Action UsersController::login() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'UsersController',
  'action' => 'login',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#2 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-11-04 17:19:15 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:19:16 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\login.ctp',
)
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 17:20:16 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:20:17 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\login.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\login.ctp',
)
Request URL: /CakePHP/users/login
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 17:21:00 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:21:00 Warning: Warning (512): Method Cake\View\Helper\SessionHelper::flash does not exist in [C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\Helper.php, line 138]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
Cake\View\Helper::__call() - CORE\src\View\Helper.php, line 138
Cake\View\Helper\SessionHelper::flash() - APP/Template\Users\login.ctp, line 2
include - APP/Template\Users\login.ctp, line 2
Cake\View\View::_evaluate() - CORE\src\View\View.php, line 972
Cake\View\View::_render() - CORE\src\View\View.php, line 932
Cake\View\View::render() - CORE\src\View\View.php, line 580
Cake\Controller\Controller::render() - CORE\src\Controller\Controller.php, line 598
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 120
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:21:00 Warning: Warning (4096): Argument 1 passed to Cake\View\Helper\FormHelper::end() must be of the type array, string given, called in C:\xampp\htdocs\CakePHP\src\Template\Users\login.ctp on line 10 and defined in [C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\Helper\FormHelper.php, line 519]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
Cake\View\Helper\FormHelper::end() - CORE\src\View\Helper\FormHelper.php, line 519
include - APP/Template\Users\login.ctp, line 10
Cake\View\View::_evaluate() - CORE\src\View\View.php, line 972
Cake\View\View::_render() - CORE\src\View\View.php, line 932
Cake\View\View::render() - CORE\src\View\View.php, line 580
Cake\Controller\Controller::render() - CORE\src\Controller\Controller.php, line 598
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 120
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:22:29 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:22:29 Warning: Warning (4096): Argument 1 passed to Cake\View\Helper\FormHelper::end() must be of the type array, string given, called in C:\xampp\htdocs\CakePHP\src\Template\Users\login.ctp on line 9 and defined in [C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\Helper\FormHelper.php, line 519]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
Cake\View\Helper\FormHelper::end() - CORE\src\View\Helper\FormHelper.php, line 519
include - APP/Template\Users\login.ctp, line 9
Cake\View\View::_evaluate() - CORE\src\View\View.php, line 972
Cake\View\View::_render() - CORE\src\View\View.php, line 932
Cake\View\View::render() - CORE\src\View\View.php, line 580
Cake\Controller\Controller::render() - CORE\src\Controller\Controller.php, line 598
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 120
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:23:31 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:23:39 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:27:04 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:27:15 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:27:37 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:27:45 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:27:45 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:28:05 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:30:34 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:30:46 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:30:46 Error: [Cake\View\Exception\MissingTemplateException] Template file "Users\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Users\\index.ctp',
)
Request URL: /CakePHP/users
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\UsersController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 17:32:28 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:34:08 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:36:02 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:39:34 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:39:49 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:40:09 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:40:34 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:40:57 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:41:04 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:42:11 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:42:11 Warning: Warning (2): Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\Debugger.php:742) in [C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Network\Session.php, line 573]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
setcookie - [internal], line ??
Cake\Network\Session::renew() - CORE\src\Network\Session.php, line 573
Cake\Auth\Storage\SessionStorage::delete() - CORE\src\Auth\Storage\SessionStorage.php, line 117
Cake\Controller\Component\AuthComponent::logout() - CORE\src\Controller\Component\AuthComponent.php, line 655
App\Controller\UsersController::logout() - APP/Controller\UsersController.php, line 57
Cake\Controller\Controller::invokeAction() - CORE\src\Controller\Controller.php, line 416
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 114
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 17:44:14 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\src\Controller\UsersController.php on line 11 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
App\Controller\UsersController::beforeFilter() - APP/Controller\UsersController.php, line 11
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 18:29:17 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Event\EventManager.php on line 385 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 18:30:41 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Event\EventManager.php on line 385 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 19:00:45 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Event\EventManager.php on line 385 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 19:01:28 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Event\EventManager.php on line 385 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 19:09:20 Warning: Warning (4096): Argument 1 passed to App\Controller\AppController::beforeFilter() must be an instance of App\Controller\Event, instance of Cake\Event\Event given, called in C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Event\EventManager.php on line 385 and defined in [C:\xampp\htdocs\CakePHP\src\Controller\AppController.php, line 53]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE\src\Error\BaseErrorHandler.php, line 139
App\Controller\AppController::beforeFilter() - APP/Controller\AppController.php, line 53
Cake\Event\EventManager::_callListener() - CORE\src\Event\EventManager.php, line 385
Cake\Event\EventManager::dispatch() - CORE\src\Event\EventManager.php, line 355
Cake\Controller\Controller::dispatchEvent() - CORE\src\Event\EventDispatcherTrait.php, line 78
Cake\Controller\Controller::startupProcess() - CORE\src\Controller\Controller.php, line 478
Cake\Routing\Dispatcher::_invoke() - CORE\src\Routing\Dispatcher.php, line 109
Cake\Routing\Dispatcher::dispatch() - CORE\src\Routing\Dispatcher.php, line 87
[main] - ROOT\webroot\index.php, line 37



2015-11-04 20:30:13 Error: Fatal Error (1): Class 'AppController' not found in [C:\xampp\htdocs\CakePHP\src\Controller\OrderController.php, line 3]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 20:30:17 Error: [Cake\Error\FatalErrorException] Class 'AppController' not found
Request URL: /CakePHP/order
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(1, 'Class 'AppContr...', 'C:\\xampp\\htdocs...', 3)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 20:31:08 Error: [Cake\Controller\Exception\MissingActionException] Action OrderController::index() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'OrderController',
  'action' => 'index',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /CakePHP/order
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\OrderController))
#2 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-11-04 20:31:32 Error: [Cake\View\Exception\MissingTemplateException] Template file "Order\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Order\\index.ctp',
)
Request URL: /CakePHP/order
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\OrderController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 20:32:30 Error: Fatal Error (4): syntax error, unexpected ')', expecting '(' in [C:\xampp\htdocs\CakePHP\src\Controller\OrderController.php, line 20]
Trace:
Cake\Error\BaseErrorHandler::handleFatalError() - CORE\src\Error\BaseErrorHandler.php, line 192
Cake\Error\BaseErrorHandler::Cake\Error\{closure}() - CORE\src\Error\BaseErrorHandler.php, line 91
[main] - [internal], line ??



2015-11-04 20:32:31 Error: [Cake\Error\FatalErrorException] syntax error, unexpected ')', expecting '('
Request URL: /CakePHP/order
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Error\BaseErrorHandler.php(91): Cake\Error\BaseErrorHandler->handleFatalError(4, 'syntax error, u...', 'C:\\xampp\\htdocs...', 20)
#1 [internal function]: Cake\Error\BaseErrorHandler->Cake\Error\{closure}()
#2 {main}


2015-11-04 20:32:56 Error: [Cake\Controller\Exception\MissingActionException] Action OrderController::index() could not be found, or is not accessible.
Exception Attributes: array (
  'controller' => 'OrderController',
  'action' => 'index',
  'prefix' => '',
  'plugin' => NULL,
)
Request URL: /CakePHP/order
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(114): Cake\Controller\Controller->invokeAction()
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\OrderController))
#2 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#3 {main}


2015-11-04 20:35:36 Error: [Cake\View\Exception\MissingTemplateException] Template file "Order\index.ctp" is missing.
Exception Attributes: array (
  'file' => 'Order\\index.ctp',
)
Request URL: /CakePHP/order
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\View\View.php(577): Cake\View\View->_getViewFileName(NULL)
#1 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Controller\Controller.php(598): Cake\View\View->render(NULL, NULL)
#2 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(120): Cake\Controller\Controller->render()
#3 C:\xampp\htdocs\CakePHP\vendor\cakephp\cakephp\src\Routing\Dispatcher.php(87): Cake\Routing\Dispatcher->_invoke(Object(App\Controller\OrderController))
#4 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#5 {main}


2015-11-04 20:36:24 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:37:07 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:39:04 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:39:53 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:43:54 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:44:55 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:45:10 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:45:11 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:45:11 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:45:12 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:45:13 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:45:13 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:46:01 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


2015-11-04 20:46:09 Error: [Cake\Routing\Exception\MissingControllerException] Controller class Placeorder.ctp could not be found.
Exception Attributes: array (
  'class' => 'Placeorder.ctp',
  'plugin' => NULL,
  'prefix' => NULL,
  '_ext' => NULL,
)
Request URL: /CakePHP/placeorder.ctp
Stack Trace:
#0 C:\xampp\htdocs\CakePHP\webroot\index.php(37): Cake\Routing\Dispatcher->dispatch(Object(Cake\Network\Request), Object(Cake\Network\Response))
#1 {main}


